patroni/ha.py

```diff
@@ -737,6 +737,11 @@ def manual_failover_process_no_leader(self):
                     return None
                 return False
 
+            # in synchronous mode we shouldn't take any action if the candidate
+            # is unhealthy but our name is not in the /sync key
```

Comment:
```suggestion
            # in synchronous mode when our name is not in the /sync key
            # we shouldn't take any action even if the candidate is unhealthy 
```
If I get it right :)